pkg/snowflake/stream.go

```diff
@@ -59,6 +60,16 @@ func (sb *StreamBuilder) WithAppendOnly(b bool) *StreamBuilder {
 	return sb
 }
 
+func (sb *StreamBuilder) WithShowInitialRows(b bool) *StreamBuilder {
+	sb.showInitialRows = false
+
+	if b {
+		sb.showInitialRows = b
+	}
+
+	return sb
```

Comment:
```suggestion
	sb.showInitialRows = b
	return sb
```
are they equivalent?

Reply:
Since it's optional I don't think it has a value unless you give it a value of false...

Reply:
`func(b bool)`, `b` must be either true or false in this context

Reply:
how I read the existing code is "if b is True then set sb.showInitialRows to True, otherwise False" which is the same as `sb.showInitialRows = b`